### DISCUSSION TITLE
Help with smoothening of ebsd property

### DISCUSSION THREAD

**Discussion by aditya-shukl** [mtex-toolbox/mtex#1957] [Ask Anything] [answered]
Hi, 

Can someone help me with smoothening of a plot ,
I use the command  
```matlab
plot(ebsd('Mineralname'),ebsd('Mineralname').ss). 
```
Here `ss` is a scalar value for each pixel that I put inside the `ebsd` object. for eg. unitcell length. Now I want to make a plot which has this property smoothed as well along with grain boundaries. For now I get the attached figure. Clearly it does not look very nice. 

### Question 2:

Instead of plotting it using the command above, is it possible to plot the property ss on top of the plot given by this command below?
```matlab
plot(grains,grains.meanOrientation)
```
I try hold on but seems like the grid of ebsd and grains are different and I cannot get the desired result.

Thanks for your help
![smooth question](https://github.com/mtex-toolbox/mtex/assets/102997857/fe1a4b9e-695c-47ba-bf40-1d5a2a2360ae)

**Comment by ralfHielscher** [comment, marked as answer]
1. Currently it is not possible to fit the pixels to the smoothed grain boundaries. 
2. you can use the command [`grainMean`](https://mtex-toolbox.github.io/EBSD.grainMean.html) to compute a grain average of the property `ss`. 
```matlab
grains.prop.ss = grainMean(ebsd,ebsd.ss,grains)
plot(grains,grains.ss)
```
Ralf
